packages/back-end/test/integrations/integration-query-test.sh

```diff
@@ -0,0 +1,4 @@
+source ~/.env
+ts-node --swc ./test/integrations/integration-query-generator.ts
+gitbranch="$(git branch --no-color --show-current)"
+python3 ./test/integrations/integration-query-runner.py $gitbranch
```

Comment:
The idea being you could just diff two json files with branch names in their title... this passes the current git branch down to the python script.

Reply:
Got branches might have slashes in them or other invalid filename characters, which may or may not break things. Not a big deal to fix right now. We can deal with that when it comes up.

Reply:
I very lazily handle this in python by doing `branch_name = sys.argv[1].replace("/", "") if len(sys.argv) > 1 else ""`

There are definitely better ways to sanitize the git branch names, but for now I figured this would work and work quickly.